src/components/Public/Page/Attachment/ListLoader.vue

```diff
@@ -189,6 +188,41 @@ export default {
       type: Boolean,
       default: false,
     },
+
+    height: {
```

Comment:
Im thinking we can use one prop called previewOptions and put all of the options inside it. Otherwise we'll have to add a prop for each new option